tests/error_005_missing_dynamic_import.ts.out

```diff
@@ -1,11 +1,11 @@
-[WILDCARD]NotFound: Cannot resolve module "bad-module.ts" from "[WILDCARD]/tests/error_005_missing_dynamic_import.ts"
-    at DenoError ([WILDCARD]/js/errors.ts:[WILDCARD])
-    at maybeError ([WILDCARD]/js/errors.ts:[WILDCARD])
-    at maybeThrowError ([WILDCARD]/js/errors.ts:[WILDCARD])
-    at sendSync ([WILDCARD]/js/dispatch.ts:[WILDCARD])
-    at codeFetch ([WILDCARD]/js/os.ts:[WILDCARD])
-    at resolveModule ([WILDCARD]/js/compiler.ts:[WILDCARD])
-    at moduleNames.map.name ([WILDCARD]/js/compiler.ts:[WILDCARD])
-    at resolveModuleNames ([WILDCARD]/js/compiler.ts:[WILDCARD])
-    at compilerHost.resolveModuleNames ([WILDCARD])
-    at resolveModuleNamesWorker ([WILDCARD])
+[WILDCARD]NotFound: Cannot resolve module "bad-module.ts" from "[WILDCARD]tests/error_005_missing_dynamic_import.ts"
+    at DenoError (deno/js/errors.ts:[WILDCARD])
+    at maybeError (deno/js/errors.ts:[WILDCARD])
+    at maybeThrowError (deno/js/errors.ts:[WILDCARD])
+    at sendSync (deno/js/dispatch.ts:[WILDCARD])
+    at codeFetch (deno/js/os.ts:[WILDCARD])
+    at resolveModule (deno/js/compiler.ts:[WILDCARD])
+    at deno/js/compiler.ts:[WILDCARD]
+    at resolveModuleNames (deno/js/compiler.ts:[WILDCARD])
```

Comment:
wasn't the [WILDCARD] here to support tests which don't have deno in a directory called deno?

Reply:
Good catch - fixed.